packages/orama/src/components/sorter.ts

```diff
@@ -170,35 +168,23 @@ async function sortBy(sorter: Sorter, docIds: [string, number][], by: SorterPara
     throw createError('UNABLE_TO_SORT_ON_UNKNOWN_FIELD', property, sorter.sortableProperties.join(', '))
   }
 
-  const docIdsLength = docIds.length
-
-  // Calculate how many documents aren't inside the sorter index.
-  // Used only for "DESC" sort.
-  let unsortableDocumentTotal = 0
-  if (isDesc) {
-    for (let i = 0; i < docIdsLength; i++) {
-      if (typeof s.docs[docIds[i][0]] === 'undefined') {
-        unsortableDocumentTotal++
-      }
+  docIds.sort((a, b) => {
+    const posA = s.docs[a[0]]
+    const posB = s.docs[b[0]]
+    if (typeof posA === 'undefined' && typeof posB === 'undefined') {
```

Comment:
We're duplicating the `typeof` operation for both `posA` and `posB`. I'd move them in a single, separate constant